sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -991,6 +999,16 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
             Row(null, null, null, "]") :: Nil
         )
 
+        // Check if corrupt records are dropped.
```

Comment:
It's not dropped, but set to null right? I'm not sure how to set JSON options in SQL string, but by default it's PERMISSIVE, so having null field looks reasonable for me.